### DISCUSSION TITLE
same query, 1 can run in script and 1 cannot run in store procedure

### DISCUSSION THREAD

**Discussion by Pierrelai99** [dbeaver/dbeaver#15744] [Q&A]
my problem is when I create a new script, and run query:
select*
from daily_stake_diff_list a
where 1=1 
and fdate<DATE_FORMAT('2022-01-16', '%Y-%m-%d')

then have result: 
![image](https://user-images.githubusercontent.com/99549872/157043161-dee70488-e744-4b2c-afe4-90f2c5f3df4c.png)

but when I create a sp:

CREATE PROCEDURE pancakeswap.spg_update_daily_stake_diff_list_v2(fdate datetime)
BEGIN

select*
from daily_stake_diff_list a
where 1=1 
and fdate<DATE_FORMAT('2022-01-16', '%Y-%m-%d') ;

END
then the result is not data:
![image](https://user-images.githubusercontent.com/99549872/157043418-1fc6dc07-e876-45f7-94c3-b17e9af10f34.png)

I think this is a bug, with same query sp cannot show true result

**Comment by LonwoLonwo** [comment]
Hello @Pierrelai99 

What database do you use?
